velox/exec/Exchange.h

```diff
@@ -103,9 +103,6 @@ class Exchange : public SourceOperator {
   /// there are more splits available or no-more-splits signal has arrived.
   ContinueFuture splitFuture_{ContinueFuture::makeEmpty()};
 
-  // Reusable result vector.
-  RowVectorPtr result_;
```

Comment:
This is needed for result reusing, otherwise we would need to reallocate and reinitialize the memory for every batch.